concurrent.ForkJoinPool.getUncaughtExceptionHandler()\nReturns the handler for internal worker threads that terminate\n due to unrecoverable errors encountered while executing tasks.\n